package.json

```diff
@@ -51,6 +51,7 @@
     "@heroku/foreman": "2.0.2",
     "@sentry/node": "5.18.1",
     "accounting": "0.4.1",
+    "algoliasearch": "^4.10.3",
```

Comment:
If there's concern with adding this dependency (along with all of its direct dependencies), it looks like we could do something like this via `@algolia/client-search`:

```typescript
import { generateSecuredApiKey } from "@algolia/client-search"

const options = {
  restrictIndices: config.ALGOLIA_RESTRICT_INDICES,
  validUntil: Math.floor(Date.now() / 1000) + 3600,
  userToken: userID
}

return generateSecuredApiKey()(
  config.ALGOLIA_SEARCH_API_KEY!,
  options
)
```

Reply:
I went ahead and remove the `ALGOLIA_APP_ID` and `ALGOLIA_ADMIN_API_KEY` config variables as they're not needed to generate a secured API key from a parent search API key but I left the `algoliasearch` library import as it's the top-level, documented entrypoint.

Reply:
I don't know that there's 'concern' per se, but I suppose any way to use and import a smaller and more focused set vs the larger monorepo lib is probably preferable.